plugins/apache_druid_logs.yaml

```diff
@@ -0,0 +1,217 @@
+version: 0.0.1
+title: Apache Druid
+description: Log parser for Apache Druid
+parameters:
+  - name: log_directory
+    description: Absolute filepath of the folder which contains the Apache Druid logs
+    type: "string"
+    required: true
```

Comment:
Generally we do not use a directory parameter, instead we just specify full paths for things like `broker_log_filename`.

I had trouble with this initially because I forgot to add a trailing `/` to my directory parameter. The collector runs fine but does not collect logs because the `/` splitting the directory and file name are missing.

I think we could remove the `log_directory` parameter and rename the filename parameters as `broker_log_path` and so on.

I do like the directory idea though. It makes it easier for the user because they just need to know where their Druid install is located. I think we could build some sort of `path.Join` function in the template, but I am not super familiar with how that works. It would allow the following cases to be valid
```
# directory | file
/logs | broker.log
/logs/ | broker.log
/logs/ | /broker.log
```

Reply:
Yeah, I just thought it would be less of a pain for the user to only have to specify the full path once rather than up to 6 times, for each enabled Druid service. Especially since I haven't found any way to configure Druid to put the log files for distinct services into distinct directories.

I initially tried to mitigate any confusion over the path being required to have a trailing `/` by manually modifying apache_druid_logs.md to have the example in there explicitly have the `/`, but the automated tests were unfortunately adamant that I not do anything besides run the automated docgen script. Then I just forgot to reconsider the design once it was replaced with `$LOG_DIRECTORY`. It was pretty fragile to being with though, so I'll either just ditch the log_directory param as suggested or see if I can figure out some sort of `join` function like you described.